torch/__init__.py

```diff
@@ -840,6 +841,28 @@ def sym_min(a, b):
         return builtins.min(a, b)
 
 
+def sym_sum(args):
+    """
+    N-ary add which is faster to compute for long lists than iterated binary
+    addition.  Only does something special for integers.
+    """
+    if overrides.has_torch_function(args):
+        return overrides.handle_torch_function(sym_sum, args, args)
+
+    found = None
+    for a in args:
+        if not isinstance(a, (SymInt, builtins.int)):
+            return builtins.sum(args)
```

Comment:
are we worried about mixed int / symint compute ever entering sym_sum? (`torch.sym_sum(int, SymInt, SymInt)`)?

I guess `builtins.sum()` will just fall back to doing a bunch of individual `__add__` calls. We could filter out the args for the symints and do a sum just on those? (just a suggestion, feel free to ignore)

Reply:
yes, it will fall back to `__add__` calls. Mixed int/symint compute is fine, we keep going. It would be a mixed int/float situation. But I do not care about optimizing it.